client/js/components/procedure/admin/AdministrationCustomFieldsList.vue

```diff
@@ -0,0 +1,401 @@
+<template>
+  <div>
+    <dp-inline-notification
+      class="mb-4"
+      data-cy="places:editInfo"
+      dismissible
+      :dismissible-key="helpTextDismissibleKey"
+      :message="Translator.trans('segments.fields.edit.info')"
+      type="info" />
+
+    <create-custom-field-form
+      :handle-success="isSuccess"
+      :is-loading="isLoading"
+      @save="customFieldData => saveNewField(customFieldData)">
+      <div>
+        <dp-label
+          class="mb-1"
+          required
+          :text="Translator.trans('options')" />
+        <dp-input
+          id="newFieldOption:1"
+          class="mb-2 w-[calc(100%-26px)]"
+          data-cy="customFields:newFieldOption1"
+          v-model="newFieldOptions[0]"
+          maxlength="250"
+          required />
+        <dp-input
+          id="newFieldOption:2"
+          class="mb-2 w-[calc(100%-26px)]"
+          data-cy="customFields:newFieldOption2"
+          v-model="newFieldOptions[1]"
+          maxlength="250"
+          required />
+
+        <div
+          v-for="(option, idx) in additionalOptions"
+          :key="`option:${idx}`">
+          <div class="w-[calc(100%-26px)] inline-block mb-2">
+            <dp-input
+              v-model="newFieldOptions[idx + 2]"
+              :id="`option:${newFieldOptions[idx + 2]}`"
+              :data-cy="`customFields:newFieldOption${idx + 2}`"
+              maxlength="250" />
+          </div>
+          <dp-button
+            class="w-[20px] inline-block ml-1"
```

Comment:
if done like suggested above, we can then check here something like `v-if="idx >= 2"`